content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md

```diff
@@ -0,0 +1,318 @@
+---
+Title: Redis Enterprise Software release notes 7.4.6-tba (July 2024)
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+compatibleOSSVersion: Redis 7.2.4
+description: Support for SHA-384 certificates in client authentication. rlcheck output in support packages. New parameters for optimize_shard_placement and recover database REST API requests.
+linkTitle: 7.4.6-tba (July 2024)
+weight: 72
+---
+
+​[​Redis Enterprise Software version 7.4.6](https://redis.com/redis-enterprise-software/download-center/software/) is now available!
+
+## Highlights
+
+This version offers:
+
+- Support for SHA-384 certificates in client authentication
+
+- `rlcheck` output in support packages
+
+- New parameters for `optimize_shard_placement` and `recover` database REST API requests
+
+## New in this release
+
+### Enhancements
+
+- Added support for SHA-384 certificates when using mutual TLS for client authentication.
+
+- Added [`rlcheck`]({{<relref "/operate/rs/references/cli-utilities/rlcheck">}}) output to the debug information collected in support packages.
+
+- Added a `replication` query parameter to [`GET /v1/bdbs/<uid>/actions/optimize_shards_placement`]({{<relref "/operate/rs/references/rest-api/requests/bdbs/actions/optimize_shards_placement">}}) REST API requests.
+
+- Added `shard_uid` and `shard_role` for each persistence file in the response body of [`GET /v1/bdbs/<uid>/actions/recover`]({{<relref "/operate/rs/references/rest-api/requests/bdbs/actions/recover">}}) REST API requests.
+
+#### Redis module feature sets
+
+Redis Enterprise comes packaged with several modules. As of version 7.4.2, Redis Enterprise includes two feature sets, compatible with different Redis database versions.
+
+Bundled Redis modules compatible with Redis database version 7.2:
+
+- [RediSearch 2.8.13]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.8-release-notes.md" >}})
+
+- [RedisJSON 2.6.10]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisjson/redisjson-2.6-release-notes.md" >}})
+
+- [RedisTimeSeries 1.10.12]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redistimeseries/redistimeseries-1.10-release-notes.md" >}})
+
+- [RedisBloom 2.6.12]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisbloom/redisbloom-2.6-release-notes.md" >}})
+
+- [RedisGears 2.0.20 preview](https://github.com/RedisGears/RedisGears/releases/tag/v2.0.20-m21)
```

Comment:
Please add a note - RedisGears preview will not be promoted to GA and will be removed from the next release.

Reply:
Added